server/controllers/chartsController.js

```diff
@@ -194,3 +195,13 @@ export const graphDataController = async (dataDb, userAccess, chart_id) => {
     studyTotals,
   }
 }
+
+function calculateStudySectionTargetValue(
+  studySectionTotalTarget,
+  studySectionTotalCount
+) {
+  if (studySectionTotalTarget > studySectionTotalCount)
+    return studySectionTotalTarget
+  if (!studySectionTotalTarget) return studySectionTotalCount
+  return 0
```

Comment:
This is a nit but I think it'll be more readable to have the guard clause first. Also, I think we want to return the `studySectionTotalCount` if we don't have a `studySectionTotalTarget`. Does that sound right to you?

```suggestion
  if (!studySectionTotalTarget) return studySectionTotalCount

  if (studySectionTotalTarget > studySectionTotalCount)
    return studySectionTotalTarget
  
  return studySectionTotalCount || 0
```

Reply:
yes that sounds right to me.

What do you think about this line `return studySectionTotalCount || 0` becoming this line `return studySectionTotalCount` otherwise I don't know when we would default to 0

Reply:
ok as long as we will always have a `studySectionTotalCount` then that will work.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/198/commits/d2d0b6fef4ed39c1fe5c13ec2e99b126130b8482